com.quexten.Goldwarden.yml

```diff
@@ -0,0 +1,56 @@
+id: com.quexten.Goldwarden
+runtime: org.gnome.Platform
+runtime-version: '45'
+sdk: org.gnome.Sdk
+command: main.py
+finish-args:
+  # Allow network access for sync
+  - --share=network
+
+  # GUI
+  - --share=ipc
+  - --socket=wayland
+  - --socket=fallback-x11
+  - --device=dri
+
+  # Lock on screensave
+  - --talk-name=org.gnome.ScreenSaver
+  - --talk-name=org.freedesktop.ScreenSaver
+
+  # Home directory access to setup browser ipc
+  - --filesystem=home
```

Comment:
which browser IPC?

Reply:
The application can emulate Bitwarden's browser IPC feature to support biometric unlock in web browsers. For browser IPC, a manifest in `~/.mozilla/native-messaging-hosts`, `~/.config/google-chrome/NativeMessagingHosts/`, etc. is necessary. Adding only these paths is problematic because then the set of supported browsers is limited to the ones noted in the yml file. Currently the app can recursively search for these directories for setup.

I suppose using a portal might also be possible here, but I will switch this to a limited set of known browsers for now.

Reply:
Okay, I rewrote some parts of the app, so that the sockets required for ssh-agent support, and for an unsandboxed cli client to communicate with the daemon inside the flatpak, can run over ~/.var/app/com.quexten.Goldwarden/data/. Since browser biometric setup is not implemented for the flatpak at the moment anyways, this allows me to completely drop the filesystem permission.